app/controllers/optimization_controller.rb

```diff
@@ -24,19 +24,12 @@ def breadcrumbs_options
     {
       :breadcrumbs => [
         {:title => _('Overview')},
-        bc_optimization,
+        {:title => _("Optimization"), :url => url_for_only_path(:action => 'show_list', :id => nil)},
```

Comment:
:+1:

Reply:
You wil need to undo this change, https://github.com/ManageIQ/manageiq-ui-classic/pull/6041 changes how optimization does breadcrumbs

Reply:
It depends what will be merged sooner... 😏 

I will undo it!

Reply:
Thanks, and if you can rebase, I'll try to test this one tomorrow :)

Reply:
Yeah, I’ll do it tomorrow.